Slavahaim/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+EXIT_STATE = 'exit!'
+ERROR_MESSAGE = 'String can not be blank'
+def get_middle(inputed_string)
+  last_element = inputed_string.size - 1
+  middle_element = inputed_string.size / 2
+  inputed_string[last_element / 2..middle_element]
```

Comment:
There is a magic numbar here. It is 2. I mean we can extract `2..middle_element` into the varible with `2..`

Reply:
Let's rename `middle_element` in `middle_range`